index.js

```diff
@@ -1,11 +1,45 @@
 const express = require('express');
 const bodyParser = require('body-parser');
 const { bookRoutes, errorRoutes, customerRoutes } = require('./routers');
+const { bookDB, cartDB } = require('./db');
 const app = express();
 const port = 8080;
 
 app.use(bodyParser.json());
 
+// user wants to buy a book
```

Comment:
Here we are! This is a great example why do we need splitting parts of the code.
You have a comment here which explains what is done here. And this is  kind of a sign to move things to a separate piece of code, in this case it should be a function.
At the moment your approach requires a reader to spend more time while reviewing.
Could you please move handler part into separate function and use it here? If you do it correct, the comment won't be necessary.

Reply:
It's funny you point this out because as soon as I finished writing my function out I started to see the need for creating the service layer abstraction. I'll be sure to have this reorganized in another commit.

Reply:
Yeah, thanks for pointing me that it's still a draft!